content/syllabuses/web-dev-boot.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Web development boot camp
+---
+ - {{% contentlink "topics/free-code-camp/javascript-data-structures-and-algorithms/basic-javascript" %}}
```

Comment:
It would be good to have another topic on:

how-to-be-successful-at-bootcamp
- tell them the plan and mechanisms
- tell them what we are looking for
- tell them how to ask for help